tests/storage/snapshot.go

```diff
@@ -312,10 +301,13 @@ var _ = SIGDescribe("[Serial]VirtualMachineSnapshot Tests", func() {
 						&expect.BExp{R: console.PromptExpression},
 						&expect.BSnd{S: "echo $?\n"},
 						&expect.BExp{R: console.RetValue("1")},
-						&expect.BSnd{S: fmt.Sprintf("%s | grep \"%s\"\n", journalctlCheck, expectedThawOutput)},
-						&expect.BExp{R: console.PromptExpression},
-						&expect.BSnd{S: "echo $?\n"},
-						&expect.BExp{R: console.RetValue("1")},
+						// always thaw
```

Comment:
it might call thaw :) since we do the freeze only if we have volume backups but we thaw if we have guest agent

Reply:
maybe use new hasGuestAgent flag here then?